crt_portal/static/js/set_referrer_on_report.js

```diff
@@ -0,0 +1,19 @@
+(function(root) {
+  referrer = document.referrer;
+  host = document.location.host;
+  if (referrer.indexOf(host) > -1) {
+    referrer = '';
+  }
+
+  if (localStorage.getItem('referrer') == null && referrer != '') {
+    localStorage.setItem('referrer', referrer);
+  }
+
+  if (localStorage.getItem('referrer') != null && localStorage.getItem('referrer') != '') {
```

Comment:
Could you use 

```if (localStorage.getItem('referrer'))``` (or the variable you declare) instead here? You can check for general truthiness.